src/main/java/cubicchunks/server/CubeProviderServer.java

```diff
@@ -403,6 +405,18 @@ private void populateCube(Cube cube) {
                 popcube.setPopulated(true);
             }
         });
+        
+        if (CubicChunks.Config.BoolOptions.USE_VANILLA_CHUNK_WORLD_GENERATORS.getValue()) {
```

Comment:
Compatibility generator pregenerate cubes in a range of 2 (cube pos.) to +x and +z.